common/model-views.cpp

```diff
@@ -1951,13 +1911,31 @@ namespace rs2
         return results;
     }
 
-    std::vector<stream_profile> subdevice_model::get_selected_profiles()
+    bool is_resolution_max(const std::vector<std::pair<int, int>>& res_values, int width, int height, const std::pair<int, int>& calibration_res)
```

Comment:
Specify the assumptions - the vector seem to be sorted.
For review - should be simplified

Reply:
comments added - let's review it

Reply:
get_max_color_resolution in progress

Reply:
done